### ISSUE TITLE
Chart prop 'zoomType' not working with Highcharts 10.2.1

### ISSUE THREAD

**Issue by veloek** [whawker/react-jsx-highcharts#376]
#### Bug or Feature Request?
Bug

#### Description
Enabling zoom with `zoomType="x"` on the `Chart` component in a `HighchartsStockChart` broke with Highcharts version 10.2.1. Haven't been able to reproduce the issue with vanilla Highcharts.

#### How to reproduce
Run the demo. Try downgrading to Highcharts version 10.2.0 and verify that the issue is gone.

#### Live demo demonstrating bug
https://stackblitz.com/edit/react-ceq9uc?file=src/MyChart.js

#### Versions
* React JSX Highcharts version: 5.0.0
* Highcharts version: 10.2.1
* React version: 18.2.0

**Comment by anajavi** [comment]
Highcharts deprecated zoomType and it somehow also broke at the same time.

Workaround is to use the new zooming prop:
https://stackblitz.com/edit/react-bp4fmf?file=src%2FMyChart.js

**Comment by veloek** [comment]
@anajavi Thanks for the workaround. It seems like something broke in Highchart's `update` method: https://jsfiddle.net/3b1eqpkw/

**Comment by anajavi** [comment]
Maybe we should have a new element for zooming, for example:
```jsx
<Zooming type="x" singleTouch={true} />
```

**Comment by veloek** [comment]
Yeah, I think that'd be nice. And perhaps a `Panning` component as well?

**Comment by anajavi** [comment]
This was fixed sometime in highcharts v11
